src/routers/products.ts

```diff
@@ -33,6 +35,6 @@ const router = createRouter();
 router.get('/', async (req, res) => {
   const response = await ProductTypesDb.getProductTypes();
   res.status(StatusCode.OK).send(JSON.stringify(response));
-});
+}, authorized());
```

Comment:
Is this correct? Doesn't this mean no one is authorized?

Reply:
Yep. I talked to brad about it and he said since nothing is using these apis currently, its better to limit permissions and change it later when we do use it